notebooks/transport/http.go

```diff
@@ -16,9 +17,19 @@ import (
 )
 
 const (
-	prefixNotebooks = "/api/v2private/flows"
-	errMissingParam = "url missing %s"
-	errInvalidParam = "url %s is invalid"
+	prefixNotebooks = "/api/v2/notebooks"
```

Comment:
I don't see this prefix used anywhere, is it used elsewhere in an unchanged file?